src/docs/patterns/lists.mdx

```diff
@@ -60,7 +197,142 @@ Occasionally, you may want users to be able to re-order the items in a list manu
+        'https://stackblitz.com/github/pxblue/angular-design-patterns?file=src%2Fapp%2Fpages%2Flist%2Faction-list%2Faction-list.component.ts'
+    }
+    react={
+        'https://codesandbox.io/s/github/pxblue/react-design-patterns/tree/master?file=/src/pages/lists/action-list/index.tsx&initialpath=action-list'
+    }
+    ionic
+    reactNative
+/>
+
+## Global List Actions
+
+Global list actions are the actions that manipulate the entire list, rather than treating each list item individually. The buttons typically sit in the [Appbar](/patterns/appbar), the panel header, or a button row (desktop-only).
+
+<ImageGrid
+    images={[GlobalActionAppBarImage, GlobalActionPanelHeaderImage, GlobalActionAsButtonImage]}
+    caption={[
+        `1: Global actions in the Appbar.`,
+        '2: Global actions in the panel header.',
+        '3: Global actions in its own button row.',
+    ]}
+/>
+
+Common global list actions include:
+
+-   Adding a new list item
+-   Sorting and filtering the list
+-   Toggling a different view, such as a grid view
+-   Exporting or downloading the list data
+
+### Drag and Drop
+
+<ImageGrid images={[SortableListImage]} regularWidth />
+<DemoCard repository={'sortable-list'} angular={'https://stackblitz.com/github/pxblue/angular-design-patterns?file=src%2Fapp%2Fpages%2Flist%2Fsortable-list%2Fsortable-list.component.ts'} react={'https://codesandbox.io/s/github/pxblue/react-design-patterns/tree/master?file=/src/pages/lists/sortable-list/index.tsx&initialpath=sortable-list'} ionic reactNative />
+
+Drag and drop is a special case for list sorting. It re-orders the items in a list manually. This should be accomplished by placing a drag handle as the leftmost element in the row. The ability to re-order the list should be locked behind an Edit Mode toggle mechanism.
+
+## Local List Actions
+
+<ImageGrid
+    images={[LocalListActionsImage]}
+    caption={[
+        '1: A more_vert icon, used to open a menu for more list actions.',
+        '2: A chevron, usually opens up a new page with list item details.',
+        '3: A few icons that serve as list action buttons.',
+        '4: List actions can be used together.',
+        '5: An inline checkbox to toggle a status.',
+        '6: An inline text field for data entry.',
+    ]}
+    gridImageProps={{ sm: 8 }}
+/>
+
+If there are actions that you would like to perform on individual list items, you may add interactive elements such as icons or a 3-dot action menu. Depending on the space available, you typically will want to use the action menu if you have more than three actions that can be taken.
+
+> If your list item is clickable for the entire item, then you should not arrange any extra local list actions, as the two interactions can intervene with each other.
+
+### Navigating Down a Tree Structure
+
+<ImageGrid
+    images={[NavigateListImage]}
+    caption={['1: A list used for separated pages.', '2: A list displaying a folder structure.']}
+    regularWidth
+/>
```

Comment:
I find the second image difficult to read. Is this something that we are encouraging teams to do? Is this common?

Reply:
Is that a problem with the divider? Is this slightly better? Or is it about indentations?

![image](https://user-images.githubusercontent.com/8997218/93806675-83913380-fc17-11ea-8720-d15dbf509a89.png)

The example is retrieved and sanitized from PredictPulse.

Reply:
I think PredictPulse is the only project I can find that involves a file tree like this.

Reply:
I think it was more of an indentation thing. If I squint my eyes everything looks like it's the same level.

Reply:
Bumped the indentation up. Now it not only has the same indentation as our drawer component, the folder icon also lines up with its parent folder's text. Hooray!